pocs/tests/test_mount.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from pocs.mount import create_mount_from_config
+from pocs.utils.location import create_location_from_config
+
+
+def test_bad_mount_port(config):
+    conf = config.copy()
+    conf['mount']['serial']['port'] = '/dev/'
+    site_details = create_location_from_config(conf)
+    with pytest.raises(SystemExit):
```

Comment:
Since we are now creating the mount outside the `Observatory` we don't need to exit, so this should become just `error.MountNotFound`. See above. Also need to import `error`.

```suggestion
    with pytest.raises(error.MountNotFound):
```